### PR TITLE
Add justifications for October 2023

### PR DESCRIPTION
**Pull request by daniel-beck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -114,6 +114,16 @@ NVD: **CWE-269: Improper Privilege Management**
...
 
 
+## [CVE-2022-25180](https://nvd.nist.gov/vuln/detail/CVE-2022-25180)
+
+### CVE Assignents
+
+Jenkins Project: **CWE-522: Insufficiently Protected Credentials**  
+NVD: **CWE-319: Cleartext Transmission of Sensitive Information**
+
+The password is never transmitted in plain text, that's not the problem.
```
Comment (yaroslavafenkin) [inline README.md:124]: Perhaps we should briefly mention what the problem is here to have higher chance of convincing them?

File: README.md [hunk 2]
```diff
@@ -250,6 +272,18 @@ NVD: **CWE-120: Buffer Copy without Checking Size of Input ('Classic Buffer Over
 Size of input is checked, but incorrectly.
 
 
+## [CVE-2022-30966](https://nvd.nist.gov/vuln/detail/CVE-2022-30966)
+
+### CWE Assignments
+
+Jenkins Project: **CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')**  
+NVD: **CWE-116: Improper Encoding or Escaping of Output**
```
Comment (yaroslavafenkin) [inline README.md:280]: 🤯 how is that about encoding

File: README.md (hunk 1 above)
Comment (daniel-beck) [reply README.md:124]: Unsure. If they've read the CVE description and advisory, it's unclear to me how they got to 319, so more explanation seems unnecessary for now.

**Review by yaroslavafenkin — APPROVED** [review]

**Review by Kevin-CB — APPROVED** [review]
Nothing to add, your justification seems good enough to me.

**Pull request merged**